README.md

```diff
@@ -61,6 +61,39 @@ sw  $t3, 8($t0)    # the_list[i+1] = $t3
     * .ascii "Hello" does not null terminate (when string ends is not specified), so please use .asciiz
     * You cannot use ALU operations (arithmetic, logic) on memory, you must load them into registers first, then store the results
     * When doing comparisons, you usually want the exact opposite e.g. say you want if (x > 2) {...} you want to jump after the block on condition !(2 < x) (slt then bne)
+        * Example
+
+Python
+```python
+if x <= y:
+    # Do something
+else:
+	# Do something else
+```
+MIPS
+```
+	# Given that x,y are both defined in the text segment
+	lw $t0, x
+	lw $t1, y
+	
+# Check comparison
+
+	# (x <= y) == !(y < x) 
+	slt $t2, $t1, $t0
+	bne $t2, $0, endif
+OR
+	# (x <= y) == (x < y + 1), given that both are integers
+	addi $t1, $t1, 1
+	slt $t2, $t0, $t1
+	beq $t2, $0, endif
```

Comment:
1. This is not faithful and probably not optimised.
2. It is definitely confusing being a fundamental example
3. It makes no sense in code which defeats the purpose of it being in a single code block

Reply:
i.e. please remove